Source/Core/VideoCommon/NetPlayChatUI.cpp

```diff
@@ -0,0 +1,112 @@
+// Copyright 2019 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#include "VideoCommon/NetPlayChatUI.h"
+
+#include <imgui.h>
+
+constexpr float DEFAULT_WINDOW_WIDTH = 220.0f;
+constexpr float DEFAULT_WINDOW_HEIGHT = 400.0f;
+
+constexpr size_t MAX_BACKLOG_SIZE = 100;
+
+std::unique_ptr<NetPlayChatUI> g_netplay_chat_ui;
```

Comment:
Is this used?

Reply:
Looks unused to me. Guessing this was moved down to `Renderer` as member.